packages/tailwindcss/theme.css

```diff
@@ -421,6 +421,13 @@
   --line-height-9: 2.25rem;
   --line-height-10: 2.5rem;
 
+  /* 3D perspectives */
+  --perspective-dramatic: 100px;
+  --perspective-near: 300px;
+  --perspective-normal: 500px;
+  --perspective-midrange: 800px;
+  --perspective-distant: 1200px;
```

Comment:
Makes sense that we need to add these to the theme if we want a named scale but also a bummer because we worked so hard to cut back on the size of the default theme. Anyone have thoughts on just making this purely numeric/bare-value based? I'm like 75% for named, 25% for numeric but curious what others think. Admittedly this is a really good example of a property where people would benefit for a small curated set of options, because nobody knows what the hell these numbers mean.

Reply:
I like the named scale as well. It's conceptually distance from the screen so bare values seems fine to me too but I think a small, default set of values is more beneficial.

Reply:
Based on our discussion in https://github.com/tailwindlabs/tailwindcss/pull/10982, I also think this small set of named values would be very helpful here, plus allowing for numeric in case someone wants to do `perspective-[3000]` (or `perspective-3000` if you're dropping brackets as part of v4).

Reply:
Thanks, @brandonmcconnell, that helped me spot that we weren't supporting bare values (e.g. `perspective-123`), just arbitrary values (e.g. `perspective-[123px]`). Both are now supported.